packages/playground/src/utils/load_deployment.ts

```diff
@@ -156,16 +149,17 @@ export async function loadK8s(grid: GridClient) {
   const failedDeployments: FailedDeployment[] = [];
 
   const promises = clusters.map(async (name, index) => {
-    const contracts = await grids[index].k8s.getDeploymentContracts(name);
-    const nodeIds = await grids[index].k8s._getDeploymentNodeIds(name);
-    const clusterPromise = grids[index].k8s.getObj(name);
-    const timeoutPromise = new Promise((resolve, reject) => {
-      setTimeout(() => {
-        reject(new Error("Timeout"));
-      }, 10000);
-    });
+    const contracts = await grids[index].k8s.getDeploymentContracts(name).catch(() => []);
+    const nodeIds = await grids[index].k8s._getDeploymentNodeIds(name).catch(() => []);
 
     try {
+      const clusterPromise = grids[index].k8s.getObj(name);
+      const timeoutPromise = new Promise((resolve, reject) => {
+        setTimeout(() => {
+          reject(new Error("Timeout"));
+        }, 10000);
```

Comment:
can we have that timeout somewhere in the code, so everyone uses that for any operation?

Reply:
@xmonader sure we can even set it as env variable